### ISSUE TITLE
New analytics.js from Google. Support?

### ISSUE THREAD

**Issue by justin808** [bgarret/google-analytics-rails#16]
Should the new analytics.js from Google be supported? (without using GA.script_source): https://developers.google.com/analytics/devguides/collection/analyticsjs/advanced
````
Advanced Configuration - Web Tracking (analytics.js)

This document provides an overview of the analytics.js collection library.

The analytics.js snippet is part of Universal Analytics, which is currently in public beta. New users should use analytics.js. Existing ga.js users should create a new web property for analytics.js and dual tag their site. It is perfectly safe to include both ga.js and analytics.js snippets on the same page.
````

**Comment by spodlecki** [comment]
PR still pending. Still a maintained Gem?

**Comment by nickhammond** [comment]
Looks like @spodlecki worked on universal support https://github.com/spodlecki/google-analytics-rails/commit/6b11d0f55ce266734f5c992adf4ae743d89a1965, nice work.

**Comment by spodlecki** [comment]
merged into master and released as `1.0.0`
